compute/instances/troubleshooting/fix-dns-routed-ipv6-only-debian-bullseye.mdx

```diff
@@ -0,0 +1,144 @@
+
+For some reason, this configuration method does not seem to play nicely with SLAAC's DNS advertisements. This results in an absence of DNS resolver configuration, thus breaking most of the network activities.
+
+Due to its modern nature and active maintenance, [`netplan` is a favorable option for configuring cloud systems](https://www.debian.org/doc/manuals/debian-reference/ch05.en.html#_the_modern_network_configuration_for_cloud) and conveniently addresses the current issue. However, in alignment with our image building policy, which aims to minimize alterations to the official upstream cloud image, `netplan` is intentionally not enabled by default in Scaleway's Debian Bullseye image.
+
+## Checking if your Instance is concerned
+
+You can check whether your Debian Bullseye Instance is concerned by running the following command, where `UUID` is the identifier of your Instance:
+
+```sh
+scw -o json Instance server get UUID | jq '.routed_ip_enabled and ([.public_ips[] | select(.family != "inet6")] == [])'
+```
+
+The command will output:
+
+* `true` if the Instance is concerned by the present procedure
+* `false` otherwise
+
+<Message type="note">
+    The above command must only be run against a Debian Bullseye Instance. It does not check for this condition itself.
+</Message>
+
+## Installing netplan
+
+The `netplan` package needs to be installed **before** you apply this procedure. If you try to apply this procedure to an Instance where `netplan` is not installed, it will fail.
+
+<Message type="tip">
+    Latest Debian Bullseye images from Scaleway ship with `netplan` pre-installed.
+</Message>
+
+### Checking for netplan
+
+To check whether your Instance has `netplan` installed, run the following command:
+    ```sh
+    dpkg-query -W netplan.io
+    ````
+The command should return an output like this, where `<VERSION>` is the currently installed version of the package, meaning you can skip directly to the first step of the procedure:
+    ```sh
+    netplan.io	<VERSION>
+    ```
+If the tool is not installed, the command will print the following:
+    ```sh
+    dpkg-query: no packages found matching netplan.io
+    ````
+In this situation, proceed with the next section to install `netplan` before applying the procedure.
+
+### Installing the tool
+
+<Message type="note">
+    All steps below require super-user (`root`) privileges.
+</Message>
+
+1. *(optional)** If, **and only if**,  your Instance is already booted using a routed IPv6-only setup, you need to temporarily configure your DNS resolver so that it can reach the Debian repositories, in order to install `netplan`. The following uses Google's DNS server:
```

Comment:
```suggestion
1. Optionally, if, **and only if**, your Instance is already booted using a routed IPv6-only setup, you must temporarily configure your DNS resolver so it can reach the Debian repositories and install `netplan`. The following uses Google's DNS server:
```

Reply:
IMO, having a specifically emphasized _(optional)_ keyword mention at the very start of the line, rather than a bare word in the sentence, allows for the optional nature of the step to stand out in the list.

Reply:
@NoSuchCommand This is international, so users can quickly scan which steps are mandatory or not, and get their desired action completed fast. If strategically it can be an issue, we can replace the word.

Reply:
That's exactly my point. Having a clear typographic emphasis helps the said quick scan. On the other hand, having this melted into the sentence as any other word makes it look like a regular step at first glance.

Reply:
@NoSuchCommand Thank you for your input. We have discussed it within the team and, for consistency purposes, we will stick with the word without brackets.